src/favouritesPage/components/favouriteCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Button, Card, LoadingSpinner } from 'hds-react';
+import { Row, Col } from 'react-grid-system';
+import { PlotSearch, PlotSearchTarget } from '../../plotSearch/types';
+import { IconCrossCircle, IconArrowDown, IconArrowUp } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+import { getInfo } from '../utils';
+import { FavouriteCardDetails } from './favouriteCardDetails';
+import FavouriteCardMap from './favouriteCardMap';
+
+interface Props {
+  target: PlotSearchTarget;
+  remove: (id: number) => void;
+  plotSearch: PlotSearch | null;
+}
+
+const FavouriteCard = (props: Props): JSX.Element => {
+  const { target, remove, plotSearch } = props;
+  const { t } = useTranslation();
+  const [fullDescription, setFullDescription] = useState(false);
+  if (!plotSearch) {
+    return (
+      <Col style={{ alignContent: 'center' }}>
+        <LoadingSpinner />
+      </Col>
+    );
+  }
+
+  const infoCols = getInfo(target, plotSearch, t);
+
+  return (
+    <Card className="favouritesPage__targetCard" border key={target.id}>
+      <Row>
+        <Col md={3} xs={4}>
+          {/* Left Column (map)*/}
+          <FavouriteCardMap target={target} />
+        </Col>
+        <Col md={9} xs={8}>
+          {/* Right Column (text and actionbuttons etc.. */}
+          <Row className="favouritesPage__targetCard__headerRow">
```

Comment:
Likewise: `FavouriteCard__header-row` (with kebab case for the subelement part).